app/assets/javascripts/components/AdminPanel/lessons_page.es6.jsx

```diff
@@ -0,0 +1,32 @@
+class LessonsPage extends React.Component {
```

Comment:
Can you pull from master and put this file under components/admin? I reorganized the folders c: